src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String string;
```

Comment:
What contains variable with name `string`? Some message or keys for launching rockets?

Reply:
It contains some string. In this it contains string "Hello mates!" -> string = "Hello mates!";

Reply:
I renamed this variable